plugins/versionpress/src/Cli/vp.php

```diff
@@ -44,11 +44,12 @@ class VPCommand extends WP_CLI_Command
      *
      * <constant>
      * : The name of the constant to set.
-     *   VP_GIT_BINARY:   Absolute path to the git binary.
-     *   VP_PROJECT_ROOT: Absolute path to the root of your project (typically
-     *                    where is the .git directory)
-     *   VP_VPDB_DIR:     Absolute path to directory where VersionPress saves
-     *                    versioned database data.
+     *   VP_GIT_BINARY:     Absolute path to the git binary.
+     *   VP_WP_CLI_BINARY:  Absolute path to the WP-CLI binary.
```

Comment:
I have just now realized that you can not use it on command line, because the command line is using the wp-cli so if it is not working properly, the command `wp vp config VP_WP_CLI_BINARY '/custom/path/to/wp-cli'` will not work. But maybe there is a way to make `vp config` work without wp-cli so you could run just `vp config VP_WP_CLI_BINARY '/custom/path/to/wp-cli'`.

But the `VP_WP_CLI_BINARY` is still useful because you can set it by manually editing wp-config.php.